sap-ha-support/xml/art-sap-ha-support.xml

```diff
@@ -0,0 +1,826 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook50-profile.xsl"
+ type="text/xml"
+ title="Profiling step"?>
+<!DOCTYPE article
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+    %entities;
+]>
+<article xml:id="article-sap-ha-support"
+         xmlns="http://docbook.org/ns/docbook"
+         version="5.0"
+         xml:lang="en"
+         xmlns:xi="http://www.w3.org/2001/XInclude"
+         xmlns:xlink="http://www.w3.org/1999/xlink">
+  <title>Supported &ha; solutions by &productname;</title>
+  <info><date>
+<?dbtimestamp format="B d, Y" ?></date>
+    <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+      <dm:bugtracker>
+        <dm:url>https://bugzilla.suse.com/enter_bug.cgi</dm:url>
+        <dm:product>??</dm:product>
+        <dm:component>??</dm:component>
```

Comment:
Product/Component is ?? right now as I don't know if we'll add a new generic Documentation component or use the SAP docs one.

Reply:
I've opened a ticket to create a new component under the Documentation product. Once that exists, I'll update this part and the PR will be ready to merge.